CHANGELOG.md

```diff
@@ -2,6 +2,18 @@
 
 ## Upcoming Version
 
+### Changed
+### Added
+### Fixed
+### Removed
+
+## v0.4.1
+
+### Changed
+
+ - Updated vmm-sys-util version to 0.14.0
+ - Updated criterion version to 0.6.0
```

Comment:
this is a test-only dependency, let's not mention it in the changelog

Reply:
done as well